data-raw/process_data_sets.R

```diff
@@ -272,3 +272,12 @@ pennies_resamples <-
   select(replicate, everything()) %>%
   unnest(cols = c(data))
 usethis::use_data(pennies_resamples, overwrite = TRUE)
+
+# Massachusetts traffic data 2020
+# Original Google Sheet here:
+# https://docs.google.com/spreadsheets/d/1rRQIDBBs8DoPAk012BdgDuf4V7iFNZzj3PL4vWfK_IQ/edit#gid=2001681887
+mass_traffic_2020 <- 
+  "https://docs.google.com/spreadsheets/d/e/2PACX-1vTM4fElimnYsn6uYW9yq6RcEJyD4gQPkU6u4HGuXfdDqoRnOWGKhsZbFdUyO9VTvfHrkGWotjxGrtI_/pub?output=csv" %>%
+  read_csv() %>%
+  clean_names()
```

Comment:
Look at the data types of the variables. Some variables such as `county` should be a factor, not a character. There might be more changes.

Reply:
How do we determine which variables should be factors for this dataset? Would community, rural_urban and dir also be factors?